tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -30,14 +32,25 @@ def bin_path(self):
 
 def file_list_to_args_for_cat_with_snapshot_subjects_and_output_file(snapshot):
   return SnapshottedProcessRequest(args=tuple(sorted(f.path for f in snapshot.files)),
-                                   snapshots=(snapshot,))
+    snapshots=(snapshot,))
+
+def file_list_to_args_for_cat_with_snapshot_subjects(snapshot):
```

Comment:
All of the new functions in this file look unused except `process_request_from_java_sources` and the actual new tests?

Reply:
they are from subsequent commits but orphaned right now. you should see them resurface in my next commits - thanks for spotting!